src/github.com/matrix-org/dendrite/common/log.go

```diff
@@ -44,13 +67,46 @@ func SetupStdLogging() {
 	})
 }
 
-// SetupFileLogging configures the logging format to a file.
-func SetupFileLogging(logFile string, logLevel logrus.Level) {
-	logrus.SetLevel(logLevel)
```

Comment:
I have concerns over the performance implications of getting rid of this. For example, if we use `SetLevel` to set the log level to "Info", then any `logrus.Debug` calls are skipped quickly. 

We should probably use `SetLevel` to set the global log threshold as high as we can.